src/services/bitcode.rs

```diff
@@ -225,10 +225,14 @@ impl BitcodeService {
     async fn fetch_file_from_all_sources(
         &self,
         uuid: DebugId,
-        file_type: &'static [FileType],
+        file_type: FileType,
         scope: Scope,
         sources: Arc<[SourceConfig]>,
     ) -> Result<Option<Arc<CacheHandle>>, Error> {
+        sentry::configure_scope(|scope| {
+            scope.set_tag("auxdif.debugid", uuid);
+            scope.set_extra("auxdif.file", file_type.as_ref().into());
```

Comment:
might as well use a `tag` for all of this, thats also done here: https://github.com/getsentry/symbolicator/blob/326177214d323227ee605899596bdfcba529b5ec/src/types/mod.rs#L747-L773

That way you should also be able to search for it.

Reply:
my understanding is that tags are searchable, but the flipside is they're expensive.  here the file_type is either going to be "plist" or "bcsymbolmap" and never a custom name, I don't ever imagine I'd want to search for this.  The DebugId OTOH is useful to search on.  Hence my choice to not use `.set_tag()` for the file type.